index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Virtual Keyboard</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Open+Sans:300,400&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <script type="module" src="app.js"></script>
+  </body>
```

Comment:
It's a good practice to use a root node and append all elements to it instead of the HTML body.

```suggestion
<body>
   <div id="app"></div>
   <script type="module" src="app.js"></script>
</body>
```

This approach used by all modern frameworks and respects other content in HTML.